es_ES/android-release-build.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: page
+title: Prepararando una aplicación de Android para su liberación
+
+permalink: /android-release/
+---
+
+Durante el ciclo de desarrollo típico, probarás una aplicación usando `flutter run` en la
+línea de comando, los botones Ejecutar y Depurar de la barra de herramientas en IntelliJ, o ambos. Por defecto, Flutter crea una versión *de depuración* de su aplicación.
+
+Cuando esté listo para preparar una versión de *liberación* para Android, por ejemplo para
+[publicarla a Google Play Store][play], siga los pasos en esta página.
+
+* TOC Placeholder
+{:toc}
+
+## Revisa el App Manifest
+
+Revisa el archivo de [Manifiesto de la aplicación][manifest] `AndroidManifest.xml` localizado
+en el directorio `<app dir>/android/app/src/main/` y verifique si los valores son correctos,
+especialmente:
+
+* `application`: Edite la etiqueta [`application`][applicationtag] para reflejar el nombre
+final de la aplicación.
+
+* `uses-permission`: Remueva el [permiso][permissiontag] `android.permission.INTERNET`
+si su código de la aplicación no necesita acceso a Internet. 
+access. La plantilla estándar incluye esta etiqueta para permitir la comunicación entre
+Herramientas de flutter y una aplicación en ejecución.
+
+## Revisa la configuración de compilación
+
+Revisa el archivo por defecto de [Gradle build][gradlebuild] `build.gradle`
+localizado en el directorio `<app dir>/android/app/` y verifique si los valores estan correctos, especialmente:
+
+* `defaultConfig`:
+
+  * `applicationId`: Especifique el (Id de Aplicación)[appid]Specify final y único.
+
+  * `versionCode` & `versionName`: Especifique el número de versión de la aplicación interna, y
+  la cadena de número de versión. Consulte la guía de información de versión en
+  la [documentación de versiones][versions] para más detalle.
+
+  * `minSdkVersion` & `targetSdkVersion`: Especifique el nivel mínimo de API y el nivel de nivel en el que se está diseñando la aplicación. Consulte la sección de contenido de la API el las [documentación de versiones][versions] para más detalle.
+
+## Añadiendo un icono para el Launcher
+
+Cuando una nueva aplicación Flutter es creada, este tiene un icono de Launcher por defecto. Para personalizar este icono usted debería 
```

Comment:
When a new Flutter app is created, it has a default Launcher icon. To customize this icon you might want to check out the Flutter Launcher Icons package.

Cuando una nueva aplicación Flutter es creada, este tiene un icono de por defecto. Para personalizar este icono es posible que desee ver el paquete...

revisar bien este párrafo.